src/core/pageModules/SocialMediaFeedModule/SocialMediaFeedModule.stories.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/function-component-definition */
+
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+
+import { SocialMediaFeedModule } from './SocialMediaFeedModule';
+
+export default {
+  title: 'Core components/Social Media Feed module',
+  component: SocialMediaFeedModule,
+  args: {
+    anchor: 'feed',
+    title: 'Social Media Feed',
+    script:
+      '<div id="flockler-embed-17988bb2c6a09d6903ef26ee6c09d20e"></div><script src="https://plugins.flockler.com/embed/17868a952430b6b61f91b78bde8668ac/17988bb2c6a09d6903ef26ee6c09d20e" async></script>',
```

Comment:
Since the nature of this is to render unsafe script tag, we could at least disable this component by default and force the developer to notice and take the risk with it. The HCRC config could include something like ENABLE_UNSAFE_FLOCKER_COMPONENT and when it's set to true, only then the component is rendered. Otherwise a console log warning could give some instructions how to enable this component by using the configuration.